test/behaviour/graql/GraqlSteps.java

```diff
@@ -259,6 +310,148 @@ public void uniquely_identify_answer_concepts(List<Map<String, String>> answersI
         }
     }
 
+    @Then("order of answer concepts is")
+    public void order_of_answer_concepts_is(List<Map<String, String>> answersIdentifiers) {
+        assertEquals(
+                String.format("The number of identifier entries (rows) should match the number of answers, but found %d identifier entries and %d answers",
+                        answersIdentifiers.size(), answers.size()),
+                answersIdentifiers.size(), answers.size()
+        );
+
+        for (int i = 0; i < answers.size(); i++) {
+            final ConceptMap answer = answers.get(i);
+            final Map<String, String> answerIdentifiers = answersIdentifiers.get(i);
+            assertTrue(
+                    String.format("The answer at index %d does not match the identifier entry (row) at index %d", i, i),
+                    matchAnswer(answerIdentifiers, answer)
+            );
+        }
+    }
+
+    @Then("aggregate value is: {double}")
+    public void aggregate_value_is(double expectedAnswer) {
+        assertNotNull("The last executed query was not an aggregate query", numericAnswers);
+        assertEquals(String.format("Expected 1 answer, but got %d answers", numericAnswers.size()), 1, numericAnswers.size());
+        assertEquals(String.format("Expected answer to equal %f, but it was %f", expectedAnswer, numericAnswers.get(0).number().doubleValue()),
+                expectedAnswer,
+                numericAnswers.get(0).number().doubleValue(),
+                0.01);
+    }
+
+    @Then("aggregate answer is empty")
+    public void aggregate_answer_is_empty() {
+        assertNotNull("The last executed query was not an aggregate query", numericAnswers);
+        assertEquals("Aggregate answer is not empty, it has a value", 0, numericAnswers.size());
+    }
+
+    @Then("group identifiers are")
+    public void group_identifiers_are(Map<String, Map<String, String>> identifiers) {
+        for (Map.Entry<String, Map<String, String>> entry : identifiers.entrySet()) {
+            String groupIdentifier = entry.getKey();
+            Map<String, String> variables = entry.getValue();
+            groupIdentifiers.put(groupIdentifier, variables.get("owner"));
+        }
+    }
+
+    @Then("answer groups are")
+    public void answer_groups_are(List<Map<String, String>> answerIdentifierTable) {
```

Comment:
same comment here as in core :)

Reply:
Fixed in core, fix propagated to client-java